dbdump/management/commands/dbdump.py

```diff
@@ -13,6 +13,10 @@
 from django.core.management.base import BaseCommand, CommandError
 from django.conf import settings
 
+# Lets you inform alternative locations
+MYSQL_DUMP_TOOL = getattr(settings, "MYSQL_DUMP_TOOL", "mysqldump")
+POSTGRE_DUMP_TOOL = getattr(settings, "POSTGRE_DUMP_TOOL", "pg_dump")
```

Comment:
I would call this POSTGRES_DUMP_TOOL as nobody really abbreviates postgres as "postgre"